osgp/platform/osgp-secret-management/src/main/resources/logback-spring.xml

```diff
@@ -7,42 +7,7 @@ SPDX-License-Identifier: Apache-2.0
 -->
 
 <configuration>
-
-  <appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
-    <encoder>
-      <pattern>[%d{yyyy-MM-dd HH:mm:ss.SSS}] [${HOSTNAME}] [%thread] %level %logger{36}@%method:%line - %msg%n</pattern>
```

Comment:
Blijft dit logging pattern gehandhaaft?
`[%d{yyyy-MM-dd HH:mm:ss.SSS}] [${HOSTNAME}] [%thread] %level %logger{36}@%method:%line - %msg%n`

Reply:
logging pattern added in application.properties